======================================================================
ERROR: Failure: ImportError (cannot import name StringIO)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/loader.py", line 413, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/lucas/Programming/Python/cassandra/python-driver/tests/integration/__init__.py", line 11, in <module>
    from cassandra.cluster import Cluster
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/cluster.py", line 26, in <module>
    from cassandra.connection import ConnectionException, ConnectionShutdown
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/connection.py", line 10, in <module>
    from cassandra.decoder import (ReadyMessage, AuthenticateMessage, OptionsMessage,
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/decoder.py", line 20, in <module>
    from io import StringIO  # ignore flake8 warning: # NOQA
ImportError: cannot import name StringIO

======================================================================
ERROR: Failure: ImportError (cannot import name StringIO)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/loader.py", line 413, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/lucas/Programming/Python/cassandra/python-driver/tests/unit/io/test_asyncorereactor.py", line 14, in <module>
    from cassandra.connection import (PROTOCOL_VERSION,
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/connection.py", line 10, in <module>
    from cassandra.decoder import (ReadyMessage, AuthenticateMessage, OptionsMessage,
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/decoder.py", line 20, in <module>
    from io import StringIO  # ignore flake8 warning: # NOQA
ImportError: cannot import name StringIO

======================================================================
ERROR: Failure: ImportError (cannot import name StringIO)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/loader.py", line 413, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/lucas/Programming/Python/cassandra/python-driver/tests/unit/io/test_libevreactor.py", line 13, in <module>
    from cassandra.connection import (PROTOCOL_VERSION,
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/connection.py", line 10, in <module>
    from cassandra.decoder import (ReadyMessage, AuthenticateMessage, OptionsMessage,
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/decoder.py", line 20, in <module>
    from io import StringIO  # ignore flake8 warning: # NOQA
ImportError: cannot import name StringIO

======================================================================
ERROR: Failure: ImportError (cannot import name StringIO)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/loader.py", line 413, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/lucas/Programming/Python/cassandra/python-driver/tests/unit/test_connection.py", line 6, in <module>
    from io import StringIO
ImportError: cannot import name StringIO

======================================================================
ERROR: Failure: ImportError (cannot import name StringIO)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/loader.py", line 413, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/lucas/Programming/Python/cassandra/python-driver/tests/unit/test_control_connection.py", line 11, in <module>
    from cassandra.decoder import ResultMessage
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/decoder.py", line 20, in <module>
    from io import StringIO  # ignore flake8 warning: # NOQA
ImportError: cannot import name StringIO

======================================================================
ERROR: Failure: ImportError (cannot import name StringIO)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/loader.py", line 413, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/lucas/Programming/Python/cassandra/python-driver/tests/unit/test_host_connection_pool.py", line 9, in <module>
    from cassandra.cluster import Session
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/cluster.py", line 26, in <module>
    from cassandra.connection import ConnectionException, ConnectionShutdown
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/connection.py", line 10, in <module>
    from cassandra.decoder import (ReadyMessage, AuthenticateMessage, OptionsMessage,
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/decoder.py", line 20, in <module>
    from io import StringIO  # ignore flake8 warning: # NOQA
ImportError: cannot import name StringIO

======================================================================
ERROR: Failure: ImportError (cannot import name StringIO)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/loader.py", line 413, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/lucas/Programming/Python/cassandra/python-driver/tests/unit/test_marshalling.py", line 22, in <module>
    from cassandra.cqltypes import lookup_casstype
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/cqltypes.py", line 28, in <module>
    from io import StringIO  # NOQA
ImportError: cannot import name StringIO

======================================================================
ERROR: Failure: ImportError (cannot import name StringIO)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/loader.py", line 413, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/lucas/Programming/Python/cassandra/python-driver/tests/unit/test_metadata.py", line 7, in <module>
    from cassandra.metadata import (Murmur3Token, MD5Token,
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/metadata.py", line 21, in <module>
    import cassandra.cqltypes as types
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/cqltypes.py", line 28, in <module>
    from io import StringIO  # NOQA
ImportError: cannot import name StringIO

======================================================================
ERROR: Failure: ImportError (cannot import name StringIO)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/loader.py", line 413, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/lucas/Programming/Python/cassandra/python-driver/tests/unit/test_parameter_binding.py", line 6, in <module>
    from cassandra.query import bind_params, ValueSequence
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/query.py", line 12, in <module>
    from cassandra.cqltypes import unix_time_from_uuid1
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/cqltypes.py", line 28, in <module>
    from io import StringIO  # NOQA
ImportError: cannot import name StringIO

======================================================================
ERROR: Failure: ImportError (cannot import name StringIO)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/loader.py", line 413, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/lucas/Programming/Python/cassandra/python-driver/tests/unit/test_policies.py", line 12, in <module>
    from cassandra.cluster import Cluster
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/cluster.py", line 26, in <module>
    from cassandra.connection import ConnectionException, ConnectionShutdown
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/connection.py", line 10, in <module>
    from cassandra.decoder import (ReadyMessage, AuthenticateMessage, OptionsMessage,
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/decoder.py", line 20, in <module>
    from io import StringIO  # ignore flake8 warning: # NOQA
ImportError: cannot import name StringIO

======================================================================
ERROR: Failure: ImportError (cannot import name StringIO)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/loader.py", line 413, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/lucas/Programming/Python/cassandra/python-driver/tests/unit/test_response_future.py", line 9, in <module>
    from cassandra.cluster import Session, ResponseFuture, NoHostAvailable
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/cluster.py", line 26, in <module>
    from cassandra.connection import ConnectionException, ConnectionShutdown
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/connection.py", line 10, in <module>
    from cassandra.decoder import (ReadyMessage, AuthenticateMessage, OptionsMessage,
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/decoder.py", line 20, in <module>
    from io import StringIO  # ignore flake8 warning: # NOQA
ImportError: cannot import name StringIO

======================================================================
ERROR: Failure: ImportError (cannot import name StringIO)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/loader.py", line 413, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/lucas/Programming/Python/cassandra/venv/2.6/lib/python2.6/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/lucas/Programming/Python/cassandra/python-driver/tests/unit/test_types.py", line 4, in <module>
    from cassandra.cqltypes import (BooleanType, lookup_casstype_simple, lookup_casstype,
  File "/home/lucas/Programming/Python/cassandra/python-driver/cassandra/cqltypes.py", line 28, in <module>
    from io import StringIO  # NOQA
ImportError: cannot import name StringIO

----------------------------------------------------------------------
Ran 12 tests in 0.403s

FAILED (errors=12)

